src/handlers/internal-server-error-handler.ts

```diff
@@ -0,0 +1,43 @@
+import { HTTP_STATUS_CODES } from "../app.constants";
+import axios from "axios";
+import PAGES from "../constants/ipv-pages";
+import { getIpvPageTemplatePath, getErrorPageTemplatePath } from "../lib/paths";
+import ERROR_PAGES from "../constants/error-pages";
+import { ErrorRequestHandler } from "express";
+
+const hasStatus = (input: unknown): input is { status: number } =>
+  !!(input as { status: number }).status;
+
+export const serverErrorHandler: ErrorRequestHandler = (
```

Comment:
just one for my own learning, is there a reason why these are named exports instead of default exports?

Reply:
Great question - I think they'd probably be better as default exports, so I'll change them!